client/messages/en.json

```
@@ -1144,12 +1177,17 @@
   "ciDaNU": "Automatically track the first page view when the script loads",
   "KzO2sB": "Initial page view tracking enabled",
   "FcSY6x": "Initial page view tracking disabled",
+  "5zGub0": "Error Tracking",
```

Biome (2.4.12)

[error] The key 5zGub0 was already declared.

(lint/suspicious/noDuplicateObjectKeys)